testcode/lib/atari/displaylist.c

```diff
@@ -0,0 +1,69 @@
+/*
+** testprogram for ANTIC instructions as defined in "_antic.h"
+**
+** 23-Feb-2017, Christian Krueger
+*/
+
+#include <conio.h>
+#include <atari.h>
+#include <peekpoke.h>
+#include <string.h>
+
+// code is only for testing purposes, as screen and display list are not aligned
+// and jumps not set!
+
+unsigned char DummyScreen[400];
+
+void DisplayList = {
```

Comment:
This is a cc65 extension, see bullet no. 6: http://cc65.github.io/doc/cc65.html
You cannot define this as unsigned char array because 16 bit addresses are embedded in a display list too ( see "DummyScreen").
The (useful) void-initialization was the reason to change the DL-defines to explicitly "unsigned char" otherwise they would occupy 16 bit...